---
title: "XenofilteR-codes to generate results in Figure S1b"
author: "sufang"
date: "6/16/2020"
output: html_document
---
* was excuted on R 3.5.1
* X86_64_w64_ming 64-bits
* The orginial XenofilteR source code is available at `https://github.com/PeeperLab/XenofilteR`

* Raw paired-end Fastq files are avaiable at GEO under assession number GSE149496
* Clean reads processed by Partek Flow RNA-seq pipeline were aligned to human (hg38) and mouse (mm10) genomes
* bam files generated by Partek Flow pipeline were used for present analysis 
* run each pair of S1.human.bam and S1.mouse.bam one by one

```{r}
library(XenofilteR)
setwd("~/R/win-library/3.5/XenofilteR")

path <-dir (system.file(package = "XenofilteR"), "AK457", full.names = TRUE)
samples <- list.files(path = path, pattern = ".bam$", full.names = TRUE)
Sample_list <- data.frame(S1.human.bam, S1.mouse.bam)
Sample_list
bp.param <- SnowParam(workers = 2, type = "SOCK")
bp.param
XenofilteR(Sample_list, destination.folder = "./0725", bp.param = bp.param, output.names =NULL)


# Swapping the input order to obtain both filtered human and murine bam files consecutively

Sample_list <- data.frame(S1.mouse.bam, S1.human.bam)
Sample_list
bp.param <- SnowParam(workers = 2, type = "SOCK")
bp.param
XenofilteR(Sample_list, destination.folder = "./0725", bp.param = bp.param, output.names =NULL)

```

